### PR TITLE
Committing dob_mismatch screen only. Routing to be added later.

### PR DESCRIPTION
**Pull request by clewis1**
Adding screen.
Tests to be carried out during routing story.
https://companieshouse.atlassian.net/browse/IDVA3-2287

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: locales/en/stop-screen.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "dob_mismatch_main_title": "The details linked to this personal code do not match the details on our records",
+    "dob_mismatch_reasons": "This may be for a number of reasons - depending on how the identity was verified, or how the PSC's details were entered into our system. You cannot continue without fixing the issue.",
+    "dob_mismatch_check_personal_code": "Check you've entered the correct personal code",
+    "dob_mismatch_character_code": "Make sure the 11 character code you entered matches the code received when you verified your identity.",
+    "dob_mismatch_try_again_link_text": "Go back to the previous screen and try again.",
+    "dob_mismatch_check_dob": "Check the PSC's date of birth on the Companies House register",
+    "dob_mismatch_if_wrong": "If the date of birth entered into the Companies House register is wrong, you need to contact the company to",
+    "dob_mismatch_corrections_link_text": "submit the paper corrections form with the correct date of birth.",
```
Comment (eedwards0) [inline locales/en/stop-screen.json:9]: Update 'form" to "forms"

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by eedwards0 — APPROVED** [review]
Stop screen matches v9 of prototype

**Pull request merged**